x-pack/plugins/data_visualizer/public/application/file_data_visualizer/components/import_summary/failures.tsx

```diff
@@ -56,11 +57,11 @@ export class Failures extends Component<Props, State> {
         }
         paddingSize="m"
       >
-        <div className="failure-list">
+        <div css={{ maxHeight: '200px', overflowY: 'auto' }}>
           {this._renderPaginationControl()}
           {this.props.failedDocs.slice(startIndex, endIndex).map(({ item, reason, doc }) => (
             <div key={item}>
-              <div className="error-message">
+              <div css={{ color: euiThemeVars.euiColorDanger }}>
```

Comment:
Same question as above. Reformat for Emotion?

```suggestion
              <div css={css`
                color: ${euiThemeVars.euiColorDanger};
              `}>
```